database/blogPosts.ts

```diff
@@ -107,3 +107,15 @@ export async function getBlogPostsBySessionToken(sessionToken: string) {
   `;
   return blogPosts;
 }
+
+export async function getSpecialBlogPosts(id: number) {
```

Comment:
Let's change this to `getBlogPostById` to be more realistic

I'll take this over

Reply:
Done in 24971a9